### ISSUE TITLE
Determine long-term plan for OSB Organization and Space GUIDs

### ISSUE THREAD

**Issue by pmorie** [kubernetes-incubator/service-catalog#583]
The OSB API currently requires that the CF organization and space GUIDs be sent on [provision](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#provisioning) requests.  PR #577 adds these fields to provision requests by populating them with the UID of the namespace as a stop-gap.  Longer term, we should consider which use-cases exist around these fields and determine exactly how they should be handled.  I can imagine scenarios where a user would want to be able to have full control over these fields; we may want to expose them via the API.

**Comment by arschles** [comment]
This is related to #638. Adding to the 0.0.3 milestone

**Comment by arschles** [comment]
Moving to the 0.1.0 milestone, since this would be nice to have before we start claiming backward compatibility for our data types (which is scheduled in 0.1.0)

**Comment by arschles** [comment]
Dupe of #800, closing
